framework/include/meshgenerators/ParsedExtraElementIDGenerator.h

```diff
@@ -0,0 +1,48 @@
+//* This file is part of the MOOSE framework
+//* https://www.mooseframework.org
+//*
+//* All rights reserved, see COPYRIGHT for full restrictions
+//* https://github.com/idaholab/moose/blob/master/COPYRIGHT
+//*
+//* Licensed under LGPL 2.1, please see LICENSE for details
+//* https://www.gnu.org/licenses/lgpl-2.1.html
+
+#pragma once
+
+#include "MeshGenerator.h"
+
+#include "FunctionParserUtils.h"
+
+class ParsedExtraElementIDGenerator : public MeshGenerator, public FunctionParserUtils<false>
```

Comment:
docstring

Reply:
What is this?

Reply:
I guess you mean doxygen string above this? I think this will be pretty much the same as the class description in validParams. But I will add.

Reply:
it shows up in a different place. thx